src/types.jl

```diff
@@ -283,15 +286,17 @@ struct AtomData
     res_number::Int
     res_name::String
     element::String
+    atomic_number::Int
```

Comment:
Isn't this redundant with `element`? If we add it here it has to be set up properly elsewhere, which would require a lookup using the element. It might be easier to remove this and have that lookup where this is used. https://github.com/JuliaPhysics/PeriodicTable.jl could be useful here.

Reply:
I guess it is redundant. `atomic_number `is just what the value is called in AtomsBase and its a required key for the system. I'll implement the `atomic_number` function to use PeriodicTable.jl.

Reply:
Actually, I just read the docs for AtomsBase both `element `and `atomic_number `are required keys:

The only required properties to be specified of the system is the species and implementations of standard functions accessing the properties of the species, currently

Geometric information: [position](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#Base.position), [velocity](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#AtomsBase.velocity), [n_dimensions](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#AtomsBase.n_dimensions)
Atomic information: [atomic_symbol](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#AtomsBase.atomic_symbol), [atomic_mass](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#AtomsBase.atomic_mass), [atomic_number](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#AtomsBase.atomic_number), [element](https://juliamolsim.github.io/AtomsBase.jl/stable/apireference/#AtomsBase.element)
Atomic and system property accessors: getindex, haskey, get, keys, pairs

Reply:
They are required for AtomsBase, but I think they are redundant in this struct. The atomic number could be retrieved from the element at the point where it is required in the AtomsBase function.